docs: add numbered step by step to contributors' guide (#1010)
devopness/devopness: CONTRIBUTING.md

We appreciate the help!

All community contributions are helpful to our team and for the Devopness community!

## Before getting started ...
Show your appreciation for Devopness 💜! It means a lot to our team:
- ⭐ Star this repository

> - Follow [Devopness page on LinkedIn](https://www.linkedin.com/company/devopness/)

Reviewer: ```suggestion
- [Sign up](https://app.devopness.com/signup/) for a FREE Devopness account
- Follow [Devopness page on LinkedIn](https://www.linkedin.com/company/devopness/)
```

I believe it makes more sense to mention creating a Devopness account under the "Before getting started... Show your appreciation for Devopness" section.

We appreciate the help!

All community contributions are helpful to our team and for the Devopness community!

## Before getting started ...
Show your appreciation for Devopness 💜! It means a lot to our team:
- ⭐ Star this repository
- [Sign up](https://app.devopness.com/signup/) for a FREE Devopness account
- Follow [Devopness page on LinkedIn](https://www.linkedin.com/company/devopness/)

> - 📣 Share our LinkedIn posts on social networks, Discord, LinkedIn, Slack, Twitter, ...

Author: ```suggestion
- 📣 Share [our LinkedIn posts](https://www.linkedin.com/company/devopness/posts/) on social networks: Discord, LinkedIn, Slack, Twitter, ...
```

## Contributing Guidelines

### Getting Started

> 1. Make sure you can access [Devopness projects](https://app.devopness.com/projects)
> 2. [Fork the repository](https://github.com/devopness/devopness/fork) and start working on it
>   - To become familiar working with forks, please read and follow the [standard fork based workflow](https://gist.github.com/Chaser324/ce0505fbed06b947d962), which could be summarized with the following steps:
>     - Fork Devopness repository
>     - Create a branch
>     - When the changes in a branch are tested, open a pull request from your fork 
> 4. [Report a bug 🐛 or submit new ideas 🆕 💡](https://github.com/devopness/devopness/issues/new/choose)
> 5.❓ Ask questions and help other Devopness users with open [discussions](https://github.com/devopness/devopness/discussions)
> 6. Find a "[Good First Issue](https://github.com/devopness/devopness/labels/good%20first%20issue)"

Reviewer: ```suggestion
1. [Fork the repository](https://github.com/devopness/devopness/fork) and start working on it
  - To become familiar working with forks, please read and follow the [standard fork based workflow](https://gist.github.com/Chaser324/ce0505fbed06b947d962), which could be summarized with the following steps:
    - Fork Devopness repository
    - Create a branch
    - When the changes in a branch are tested, open a pull request from your fork 
2. [Report a bug 🐛 or submit new ideas 🆕 💡](https://github.com/devopness/devopness/issues/new/choose)
3.❓ Ask questions and help other Devopness users with open [discussions](https://github.com/devopness/devopness/discussions)
4. Find a "[Good First Issue](https://github.com/devopness/devopness/labels/good%20first%20issue)"
```

Complementing the suggestion made in [LINK](#), here the citation to Devopness APP would be removed
